src/utils/get-endpoints.ts

```diff
@@ -0,0 +1,15 @@
+import { HeliusCluster } from '../types'
+
+/**
+ * Retrieves the API and RPC URLs for the specified cluster
+ */
+export function getEndpoints(cluster: HeliusCluster): { api: string; rpc: string } {
+  switch (cluster) {
+    case "devnet":
+      return { api: "https://api-devnet.helius.xyz", rpc: "https://devnet.helius-rpc.com" }
+    case "mainnet-beta":
+      return { api: "https://api.helius.xyz", rpc: "https://mainnet.helius-rpc.com" }
+    default:
+      throw new Error(`Unknown cluster ${cluster}`)
```

Comment:
Previously, it would silently accept if you'd pass in `testnet` which would break the setup. It's probably better to fail fast here.

Reply:
Very nice catch!